penguin/web/services/projects.py

```
@@ -0,0 +1,355 @@
+from __future__ import annotations
+
+import asyncio
+from pathlib import Path
+from typing import Any, Dict, Optional
```

Ruff (0.15.11)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)